dali/pipeline/pipeline.cc

```diff
@@ -675,6 +676,7 @@ void Pipeline::PrepareOpSpec(OpSpec *spec, int logical_id) {
   spec->AddArg("max_batch_size", max_batch_size_)
     .AddArg("num_threads", num_threads_)
     .AddArg("device_id", device_id_)
+    .AddArg("checkpointing", checkpointing_)
```

Comment:
I think it should as well go as `AddInternalArg(...);` to op schema with a false as a default.